marimo/_plugins/ui/_impl/table.py

```diff
@@ -303,10 +313,22 @@ def search(self, args: SearchTableArgs) -> Union[JSONType, str]:
 
         # If no query or sort, return nothing
         # The frontend will just show the original data
-        if not args.query and not args.sort:
+        if not args.query and not args.sort and not args.filters:
             self._filtered_manager = self._manager
             return []
 
+        if args.filters:
+            # TODO: don't allow no polars and pandas to get here
```

Comment:
Just checking if this needs to be addressed now or if it's fine to get to later

Reply:
oh woops - i did this already, forgot to remove the todo